### PR TITLE
Manual Analysis for Linux

### PR DESCRIPTION
**Pull request by amelie-renard-sonarsource**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+
+It is very easy to manually analyze a C/C++/Objective-C project with SonarQube:
+
+. Create a `sonar-project.properties` file to store your configuration
+.. Add a projectKey of your project
+.. Add a Build Wrapper output directory
+.. https://docs.sonarqube.org/latest/user-guide/user-token/[Generate a unique token] and set it as "sonar.login"
+.. Set `sonar.host.url` to your SonarQube server URL
+. Download and unzip the Build Wrapper for Linux from {SonarQube URL}/static/cpp/build-wrapper-linux-x86.zip
+. Add the Build Wrapper to your `%PATH%`
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
. Add the Build Wrapper to your `$PATH`
```

**Review by arseniy-sonar — CHANGES_REQUESTED** [review]
Please, also add a short shell script, that would download build-wrapper and clang-analyzer and run the analysis.

File: README.adoc [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+. Create a `sonar-project.properties` file to store your configuration
+.. Add a projectKey of your project
+.. Add a Build Wrapper output directory
+.. https://docs.sonarqube.org/latest/user-guide/user-token/[Generate a unique token] and set it as "sonar.login"
+.. Set `sonar.host.url` to your SonarQube server URL
+. Download and unzip the Build Wrapper for Linux from {SonarQube URL}/static/cpp/build-wrapper-linux-x86.zip
+. Add the Build Wrapper to your `$PATH`
+. Download and unzip the https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[SonarScanner] for Linux
+. Add the bin directory of the SonarScanner to your `%PATH%`
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
. Add the bin directory of the SonarScanner to your `$PATH`
```

File: README.adoc [hunk 3]
```diff
@@ -1,3 +1,24 @@
...
+. Create a `sonar-project.properties` file to store your configuration
+.. Add a projectKey of your project
+.. Add a Build Wrapper output directory
+.. https://docs.sonarqube.org/latest/user-guide/user-token/[Generate a unique token] and set it as "sonar.login"
+.. Set `sonar.host.url` to your SonarQube server URL
+. Download and unzip the Build Wrapper for Linux from {SonarQube URL}/static/cpp/build-wrapper-linux-x86.zip
+. Add the Build Wrapper to your `$PATH`
+. Download and unzip the https://docs.sonarqube.org/latest/analysis/scan/sonarscanner/[SonarScanner] for Linux
+. Add the bin directory of the SonarScanner to your `$PATH`
+. Wrap your clean build command with the Build Wrapper. E.g.: `build-wrapper-linux-x86-64 --out-dir bw-output make clean all` (with `bw-output` being the same directory that the one set in `sonar-project.properties`)
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
. Wrap your clean build command with the BuildWrapper. E.g.: `build-wrapper-linux-x86-64 --out-dir bw-output make clean all` (with `bw-output` being the same directory as the one set in `sonar-project.properties`)
```
"BuildWrapper" for consistency with "SonarScanner"

File: README.adoc [hunk 4]
```diff
@@ -1,3 +1,24 @@
...
+. Run `sonar-scanner`
+
+You can take a look at the link:sonar-project.properties[sonar-project.properties] and link:build-analyze-project.sh[build-analyze-project.sh] (script that downloads the `sonar-scanner` and the Build Wrapper before compiling and analyzing the project ) to see it in practice. Please be aware that the SonarQube server URLs and the `sonar.login` are not completed in these examples.
```
Comment (arseniy-sonar) [inline README.adoc]:
```suggestion
You can take a look at the link:sonar-project.properties[sonar-project.properties] and link:build-analyze-project.sh[build-analyze-project.sh] (a script that downloads the `sonar-scanner` and the Build Wrapper before compiling and analyzing the project) to see it in practice. Please be aware that the SonarQube server URLs and the `sonar.login` are not completed in these examples.
```

**Review by arseniy-sonar — APPROVED** [review]
Looks good! Just some wording and can be merged

File: README.adoc (hunk 3 above)
Comment (amelie-renard-sonarsource) [reply README.adoc]: I prefer being consistent with SQ documentation that uses "SonarScanner" but "Build Wrapper" if you don't mind

**Pull request merged**